unstoppable-ios-app/domains-manager-ios/Entities/UIMenuDomainAvatarLoader.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UIMenuDomainAvatarLoader.swift
+//  domains-manager-ios
+//
+//  Created by Oleg Kuplin on 15.06.2023.
+//
+
+import UIKit
+
+struct UIMenuDomainAvatarLoader {
+    
+    @MainActor
+    static func menuAvatarFor(domain: DomainDisplayInfo,
```

Comment:
I can see a number of this literal `20` -- should be extracted if this is the same in all places

Reply:
I've checked the codebase, after the update there's just one place with number '20', default value for function argument 👍